laundry-management-service/laundrymanagement-app/src/main/java/com/microservices/laundrymanagement/configuration/OrderServiceBeansConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.microservices.laundrymanagement.configuration;
+
+import com.microservices.laundrymanagement.dto.DetailSubmissionDto;
+import com.microservices.laundrymanagement.dto.OrderSubmissionDto;
+import com.microservices.laundrymanagement.entity.OrderEntity;
+import com.microservices.laundrymanagement.entity.OrderStatus;
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.spi.MappingContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static org.modelmapper.config.Configuration.AccessLevel.PRIVATE;
+
+@Configuration
+public class OrderServiceBeansConfig {
+
+    @Bean
+    ModelMapper modelMapper() {
+        ModelMapper modelMapper = new ModelMapper();
+        modelMapper.getConfiguration()
+                .setFieldMatchingEnabled(true)
+                .setFieldAccessLevel(PRIVATE);
```

Comment:
Do you really need those configs if you use Converter?

Reply:
You are right, this config is not needed.